pkg/cmd/pipeline/describe.go

```diff
@@ -63,7 +63,8 @@ func printPipelineDescription(out io.Writer, p cli.Params, pname string) error {
 
 	pipeline, err := cs.Tekton.TektonV1alpha1().Pipelines(p.Namespace()).Get(pname, metav1.GetOptions{})
 	if err != nil {
-		return err
```

Comment:
this must be an error because user asked to display details of a pipeline **which does not exist**